module.json

```diff
@@ -21,7 +21,9 @@
     "minar": "^1.0.0"
   },
   "testDependencies": {
-    "mbed-drivers": "^1.0.0"
+    "mbed-drivers": "^1.0.0",
+    "unity": "^2.0.1",
+    "greentea-client": "^0.1.4"
```

Comment:
"^0.1.4" means "==0.1.4" for major version 0, see the "dependencies" section in http://yottadocs.mbed.com/reference/module.html. Probably not the intention here.